cmd/start.go

```diff
@@ -181,11 +181,13 @@ func runContainer(cmd *cobra.Command, args []string) {
 			if !testEmptyDir {
 				log.Fatal(getUnderlyingStorage(flavor) + " is not empty, doing nothing.")
 			}
-			if runtime.GOOS == "linux" {
-				applySeLinuxLabel(getUnderlyingStorage(flavor))
-			}
 			envs = append(envs, "OSD_PATH="+getUnderlyingStorage(flavor))
-			volumeBindings = append(volumeBindings, getUnderlyingStorage(flavor)+":"+getUnderlyingStorage(flavor))
+			if runtime.GOOS == "linux"  {
```

Comment:
You need to keep `volumeBindings` outside of the Linux condition and only add `:z` if Linux. Btw did you actually validated that adding `:z` works?